main.js

```diff
@@ -0,0 +1,203 @@
+// Getting the elements from the html page
+const inputTitle = document.querySelector('#title');
+const inputAuthor = document.querySelector('#author');
+const form = document.querySelector('form');
+const list = document.querySelector('ul');
+
+// creating the array of the objects
+const book = [];
+
+// This function acts like a constructor for the book object
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
```

Comment:
[OPTIONAL] Kindly use a unique id property for each book since it would make it easier to remove books with duplicate values. :+1: :smiley: :smiley:

Reply:
Fixed.